alfalfa_worker/Dockerfile

```diff
@@ -21,10 +21,7 @@ ENV PYTHONPATH="${HOME}:${PYTHONPATH}"
 
 COPY ./alfalfa_worker ${HOME}/alfalfa_worker
 
-RUN pip3.8 install virtualenv \
-  && pip3.8 install \
-  scipy \
-  symfit
```

Comment:
The worker is no longer providing builtin python dependencies outside of what base alfalfa needs.

Reply:
This is partially due to removing the `refrig_case` test which required `symfit`. However `symfit` doesn't support python 3.12 and the project in general seems semi-abandoned so I removed the test case and replaced it with a small office based test model.

Reply:
Not quite. https://github.com/NREL/alfalfa/wiki/How-to-Migrate-EnergyPlus-Python-Plugins you can provide a `requirements.txt` which results in a virtual environment being created for your model which has your requirements installed.

Reply:
Yes, this is staying the same, yes?  The E+ model with Python EMS + 3rd party Python modules just also needs to take responsibility for that requirements.txt.  This makes sense to me and I think it's important to continue to support this in Alfalfa.